src/Disks/DiskLocal.cpp

```diff
@@ -54,53 +51,6 @@ std::mutex DiskLocal::reservation_mutex;
 
 using DiskLocalPtr = std::shared_ptr<DiskLocal>;
 
-static void loadDiskLocalConfig(const String & name,
```

Comment:
this code is just moved to a separate file, see `src/Disks/loadLocalDiskConfig.cpp`